boards/FSM/Src/App/App_AcceleratorPedal.c

```diff
@@ -29,3 +41,50 @@ bool App_AcceleratorPedal_IsEncoderAlarmActive(
 {
     return accelerator_pedal->is_encoder_alarm_active();
 }
+
+float App_AcceleratorPedal_GetPedalPercentage(
+    const struct AcceleratorPedal *const accelerator_pedal)
+{
+    const uint32_t encoder_counter_value =
+        accelerator_pedal->get_encoder_counter_value();
+    const uint32_t encoder_max_pressed_value =
+        accelerator_pedal->encoder_max_pressed_value;
+    const float percent_deflection = 0.03f;
+
+    const float low_end_deadzone_threshold =
+        (float)encoder_max_pressed_value * percent_deflection;
+
+    // TODO: Determine the high end deadzone threshold
```

Comment:
Could we file a ticket for this and note it here?

Reply:
Should be done now 🌮